pkg/validatorfactory/factory.go

```diff
@@ -354,6 +354,7 @@ func (s *ValidatorFactory) ValidatorsForGVK(gvk schema.GroupVersionKind) (*Valid
 	}
 
 	for nam, def := range openapiSpec.Components.Schemas {
+		ApplySchemaPatches(0, gvk.GroupVersion(), nam, def)
```

Comment:
Passing `0` should work for now but in the future...

Reply:
For the version string I know there is the k8s version inside the top-level OpenAPI spec. It might take some work to associate the version with each individual `spec.Schema` instance inside the `openapiclients`, which would be necessary because `CompositeClient` merges all schemas together and could lose information about which schema came from which source.

Reply:
For now 0 is ok. Lets make an issue so we don't forget.

Reply:
[Done](https://github.com/kubernetes-sigs/kubectl-validate/issues/68)